### PR TITLE
Issue 124: Fix sitemap to output absolute URLs

### PR DESCRIPTION
**Pull request by rachidatecs**
Solves #124 

## Changes proposed in this pull request:
- Store site url in .env (locally)
- Pull site url in .eleventy.js and use it in a custom filter that prepends it to urls
- Use the custom filter in sitemap.xml.njk

## security considerations

Can we store the site url in an environment variable on production?

### CODE REVIEW

**Comment by rachidatecs** [comment]
Added environment variable FQDN = "https://beta.get.gov" on cloud.gov. Will need to edit it if beta is a temporary subdomain @neilmb @h-m-f-t

File: README.md [hunk 1]
```diff
@@ -180,6 +180,10 @@ CSS and SASS can be added or imported into the `styles/styles.scss`. You can als
 
 Javascript can be added to the admin UI or site UI by adding or importing code into the `js/admin.js` or `js/app.js` files respectively. This template uses [esbuild](https://esbuild.github.io/) to bundle your javascript and fingerprint the files in the site build.
 
+### Managing environment variables
+
+Environment variables are set and managed on [pages.cloud.gov](https://pages.cloud.gov/)
```
Comment (neilmb) [inline README.md:185]: **question (non-blocking)**: Do we want to note here that `BASEURL` is already defined by the platform and that's the one that we use?

**Review by neilmb — APPROVED** [review]
LGTM. I like that we are using the existing `BASEURL` environment variable instead of having to define our own.

File: README.md (hunk 1 above)
Comment (rachidatecs) [reply README.md:185]: Well, answer the upcoming slack invite

**Pull request merged**